src/message/message-builder.js

```diff
@@ -2,7 +2,7 @@ var C = require( '../constants/constants' ),
 	SEP = C.MESSAGE_PART_SEPERATOR;
```

Comment:
This file doesn't have any changes from what I can see, might be good to exclude it from the commit